.mega-linter.yaml

```diff
@@ -26,9 +27,8 @@ DISABLE_ERRORS_LINTERS:
   - JSON_JSONLINT
   - MAKEFILE_CHECKMAKE
   - MARKDOWN_MARKDOWN_LINK_CHECK
-  - PHP_PHPCS
-  - PHP_PHPSTAN
-  - PHP_PSALM
   - SPELL_MISSPELL
   - SQL_TSQLLINT
   - YAML_V8R
+PHP_PHPCS_ARGUMENTS: --standard=PSR12
```

Comment:
@nvuillam Sorry for tagging you. Is this the right way to set this?

```sh
phpcs --standard=PSR12 src tests
```

I searched the documentation and on GitHub, but I'm not sure......

Reply:
@eitsupi no problem :)
i think it should work, but usually it's easier to read and maintain to define one line by argument :)

```yaml
PHP_PHPCS_ARGUMENTS:
  - --standard=PSR12
```

Reply:
Thank you for using MegaLinter :)
Don't hesitate to post about it if you like it 👼